README.md

```diff
@@ -22,9 +22,20 @@ The scanner is located in its [own repository](https://github.com/google/osv-sca
 This repository contains all the code for running https://osv.dev on GCP. This
 consists of:
 
--   API server (`gcp/api`)
--   Web interface (`gcp/appengine`)
--   Workers for bisection and impact analysis (`docker/worker`)
+| directory       | what |
+|-----------------|------|
+| `deployment/`   | Terraform, Cloud Deploy & App Engine config files <br /> A few Cloud Build config yamls <br /> Old (no longer used?) `api-staging` and `api-test` Cloud Run configs  |
+| `docker/`       | CI docker files (`ci`, `deployment`, `terraform`) <br /> Workers for bisection and impact analysis (`worker`, `importer`, `exporter`, `alias`, `worker-base`) <br /> The determine version `indexer`|
+| `docs/`         | Jekyll files for https://google.github.io/osv.dev/ <br /> `build_swagger.py` and `tools.go` |
+| `gcp/api`       | OSV API server files (including files for the local ESP server) <br /> protobuf files in `/v1`|
+| `gcp/appengine` | The backend of the osv.dev web interface, with the frontend in `frontend3` <br /> Blog posts (in `blog`) <br /> App Engine Cron Handlers (to be removed) <br /> The  datastore indexes file (`index.yaml`) | 
+| `gcp/functions` | The Cloud Function for publishing PyPI vulnerabilities, unmaintained |
+| `osv/`          | The core OSV Python library, used in basically all Python services <br /> OSV ecosystem package versioning helpers in `ecosystems/` <br /> Datastore model definitions in `models.py` |
+| `tools/`        | Misc scripts/tools, mostly intended for development (datastore stuff, linting) <br /> The `indexer-api-caller` for indexer calling |
+| `vulnfeeds/`    | Go module for (mostly) the NVD CVE conversion <br /> The Alpine feed converter (`cmd/alpine`) <br /> The Debian feed converter (`tools/debian`, which is written in Python) |
+
+[#1714](https://github.com/google/osv.dev/issues/1714)
```

Comment:
No need to reference the issue here (that might just confuse users more).

Reply:
Done